docs/demos/rabi.ipynb

```diff
@@ -571,6 +571,7 @@
     "    fig.suptitle(title_label)\n",
     "    return fig, axes\n",
     "\n",
+    "\n",
```

Comment:
Was this on purpose?

Reply:
nope

Reply:
apparently it was nbqa-black doing some funny sfuf